CONTRIBUTING.md

```diff
@@ -75,9 +75,16 @@ To develop locally:
    ```
 
 3. Install yarn:
+The modern way to use Yarn without binary installation often requires `sudo` or `administrator mode`. 
```

Comment:
I have never heard of this "modern way" before.

do you have any links or references backing this up?